src/plugins/data_source_management/public/components/data_source_menu/types.ts

```diff
@@ -39,6 +39,7 @@ export const DataSourceComponentType = {
   DataSourceView: 'DataSourceView',
   DataSourceAggregatedView: 'DataSourceAggregatedView',
   DataSourceMultiSelectable: 'DataSourceMultiSelectable',
+  NoDataSource: 'NoDataSource',
```

Comment:
why exposing it here? Can we mount this component when there is no data sources?

Reply:
removed